### ISSUE TITLE
Do remove option availability tweaking for the nucleus

### ISSUE THREAD

**Issue by hhyyrylainen** [Revolutionary-Games/Thrive#2211]
as the nucleus can't be removed there organelle popup menu should not have the option to remove it (if it wasn't placed in the current editor session, a nucleus should be possible to remove if it was placed in the current editor session).

It's currently just disabled. So maybe two changes should be done:
- Hide the "delete" option entirely if the context menu is opened on an organelle that can't be removed
- The nucleus should be possible to remove during the current editor session

**Comment by 84634E1A607A** [comment]
You can always remove a nucleus in the current editor section just by clicking "Undo" (?)

**Comment by hhyyrylainen** [comment]
Yes, but for convenience we have made it so that the right click "remove" option also can do that for free, so it is inconsistent if the nucleus can't be removed that way for free but other organelles can be.

**Comment by Kasterisk** [comment]
For consistency hiding the delete button means also hiding it for the other cases which means the option to disable the delete button is redundant. Would it make sense to only just enable the delete option if the nucleus is placed in the current editor session but keep the rest the same?

**Comment by hhyyrylainen** [comment]
Actually now thinking about this again, maybe having a tooltip explaining why you can't delete an organelle when the option is disabled, would probably have the most explaining power to reduce the confusion of the players.

**Comment by Kasterisk** [comment]
Done https://github.com/Revolutionary-Games/Thrive/pull/3402.
